types/node-insights/node-insights-tests.ts

```diff
@@ -0,0 +1,25 @@
+import Insights from 'node-insights';
```

Comment:
Instead use `import x = require('module')` syntax. You can still use this style byt the `--allowSyntheticDefaultImports` or `--esModuleInterop` should be turned on instead.